.gitignore

```diff
@@ -1,3 +1,4 @@
+*.sw?
```

Comment:
This will prevent someone from accidentally committing vim swap files.  When you open a file for exiting in vi/vim, you will see a ".$FILENAME.swp" file as long as the text buffer remains open.